src/app/store/reducers/app.reducer.ts

```diff
@@ -37,6 +37,8 @@ import {
     ModelClosedAction,
     TOOLBAR_MESSAGE,
     ToolbarMessageAction,
+    LOADED_APPLICATION,
```

Comment:
LOADED_APPLICATION or LoadApplication?
Can we call them:
SET_APPLICATION_LOADER and
SetApplicationLoaderAction?